style/app.scss

```diff
@@ -547,6 +547,14 @@ a:active
   color            : #9c9c9c;
 }
 
+.graph-control-button img {
+  width: inherit;
+  height: inherit;
+  position: absolute;
```

Comment:
Similar to my comments to @leenlab2, avoid absolute positioning whenever possible. In this particular case, you should set a *margin* on the parent button component, and then width/height to `100%` here.

Reply:
What makes absolute positioning bad?

Reply:
From what I gather, it's because absolute positioning is akin to customizing which pixels the element is on rather than following the flow of the page. It might not look the same on different computers/screen sizes.

Reply:
@kenneth-miura yeah, as @leenlab2 said, elements that use absolute positioning are hard to mix with elements that don't (and the latter is the default). They're essentially two different systems of positioning, and mixing the two can lead to unexpected results, especially as screen sizes change or you zoom in/out of the page.

In the case of this button image it doesn't *really* make a difference because the image is the only child of the button, but the default should be to avoid it.

Reply:
Ok thanks, makes sense. So red flags for absolute positioing causing issues is when absolutely positioned elements are siblings to normal elements?

Reply:
Yeah, I think that's a good way to think about it. I also saw this StackOverflow post with more nuanced answers: https://stackoverflow.com/questions/186044/is-it-considered-bad-practice-to-use-absolute-positioning/187310.